crashtracker/src/collector/emitters.rs

```diff
@@ -1,12 +1,13 @@
-// Copyright 2021-Present Datadog, Inc. https://www.datadoghq.com/
```

Comment:
Silly question:  do these copyright notices indicate the year of first existence for the noted file, or the last update?

Reply:
I assume first year of existence.  At Amazon we stopped using years, and went to a SPDX header which was much cleaner